website/docs/prepare-for-merge.md

```diff
@@ -47,7 +48,7 @@ import MultidimensionalContentControlsPartial from '@site/docs/partials/_multidi
         <div className='input-container'><input id="cl-6" type='checkbox'/><span className='done'></span></div>
         <div className='guidance-container'>
             <label htmlFor="cl-6">Verify your Prysm version</label>
-            <p>Verify that you're running Prysm <code>v4.0.0</code> by issuing the following command: <code>prysm.sh beacon-chain --version</code> (Linux) <code>prysm.bat beacon-chain --version</code> (Windows).</p>
+            <p>Verify that you're running Prysm <code><PrysmVersion/></code> by issuing the following command: <code>prysm.sh beacon-chain --version</code> (Linux) <code>prysm.bat beacon-chain --version</code> (Windows).</p>
```

Comment:
Is there a way not to display the version number on a whole new line?
![image](https://github.com/prysmaticlabs/documentation/assets/4943830/aaca50ff-5ac1-4f23-8b97-f57cd0e33b05)

(Maybe it's tricky to do. In such a case, that's perfectly fine IMO to let it like that.)

Reply:
I think this was a mistake i'll fix this